ui/app/components/sidebar/nav/cluster.js

```diff
@@ -5,6 +5,7 @@
 
 import Component from '@glimmer/component';
 import { service } from '@ember/service';
+import config from 'vault/config/environment';
```

Comment:
Oh cool, so this acts as a sort of development feature flag. Good idea!

(This can be in a separate PR) but this could be formalized a bit and drawn into a helper so you don't need the getter _every_ time.

Reply:
Yeah this is definitely a new pattern to use the environment state like this. But with the new release cycle, I foresee us developing more iteratively instead of relying on sidebranches so it would make sense to make an app-wide helper!